docs/hub/models-gated.md

```diff
@@ -1,94 +1,149 @@
-notebook_login()
-```
+To enable access requests, go to the model settings page. By default the model is not gated. Click on "Enable Access request" on the top-right corner.
 
-Then, ensure that your library uses the token. This is now the case by default in huggingface-maintained libraries.
 
-However, on older versions of libraries, for example if using 🤗 Transformers with a version of `huggingface_hub` older than `v0.10` [you'll need to pass `use_auth_token=True`](https://huggingface.co/docs/transformers/main/en/main_classes/model#transformers.PreTrainedModel.from_pretrained) when calling `.from_pretrained()`.  
+<div class="flex justify-center">
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-disabled.png"/>
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-disabled-dark.png"/>
+</div>
 
-## Modifying the prompt 
+By default, access to the model is automatically granted to the user when requesting it. This is referred to as **automatic approval**. In this mode, any user can access your model once they've shared their personal information with you.
 
-The User Access request dialog can be modified to include additional text and checkbox fields in the prompt. To do this, add a YAML section to the model's `README.md` file (create one if it does not already exist) and add an `extra_gated_fields` property. Within this property, you'll be able to add as many custom fields as you like and whether they are a `text` or `checkbox` field. An `extra_gated_prompt` property can also be included to add a customized text message.
+<div class="flex justify-center">
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-enabled.png"/>
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-enabled-dark.png"/>
+</div>
 
-```
----
-extra_gated_prompt: "You agree to not use the model to conduct experiments that cause harm to human subjects."
-extra_gated_fields:
- Company: text
- Country: text
- I agree to use this model for non-commercial use ONLY: checkbox
----
-```
+If you want to manually approve which users can access your model, you must set it to **manual approval**. When this is the case, you will notice more options:
+- **Add access** allows you to search for a user and grant them access even if they did not requested it.
+- **Notification frequency** let you configure when to get notified if new users request access. It can be set to once a day or real-time. By default, an email is sent to your primary email address. You can set a different email address in the **Notifications email** field. For models hosted under an organization, emails are sent to the first 5 admins of the organization.
```

Comment:
```suggestion
- **Notification frequency** lets you configure when to get notified if new users request access. It can be set to once a day or real-time. By default, an email is sent to your primary email address. You can set a different email address in the **Notifications email** field. For models hosted under an organization, emails are sent to the first 5 admins of the organization.
```